index.css

```diff
@@ -0,0 +1,326 @@
+@import url('https://fonts.googleapis.com/css2?family=Lato:wght@700&family=Lobster&family=Roboto+Mono:wght@200;400&display=swap');
+
+*,
+*::before *::after {
+  box-sizing: border-box;
+}
+
+body {
+  font-size: 62.5%;
+  padding: 0;
+  margin: 0;
+  font-family: 'Lato', sans-serif;
+}
+
+.header {
+  display: flex;
+  align-items: center;
+  justify-content: space-around;
+  padding: 2% 0;
+}
+
+.header__svg {
+  position: relative;
+  left: 25px;
+  width: 11%;
+}
+
+.header__img {
+  width: 45%;
+  margin: auto 29%;
+}
+
+.hero {
+  background-image: url('./img/hero_container.png');
+  background-size: 100%;
+  background-repeat: no-repeat;
+  height: 33rem;
+  width: 100%;
+}
+
+.main {
+  display: flex;
+  flex-direction: column;
+  align-items: center;
+  background-image: url('./img/pattern.png');
+  background-repeat: repeat;
+  height: 100%;
+  width: 100%;
+  font-size: 1rem;
+  color: #fff;
+}
+
+.main__h2 {
+  font-size: 1.2rem;
+}
+
+.cards-container {
+  display: flex;
+  flex-direction: column;
+  padding: 1.5rem 0;
+  gap: 5px;
+}
+
+.cards {
+  display: flex;
+  align-items: center;
+  margin: auto 3%;
+  background-color: rgba(133, 133, 133, 0.3);
+}
+
+.main__h3 {
+  color: #ff5438;
+}
+
+.cards__svg {
+  width: 20%;
+  margin: 0 5%;
+}
+
+.cards__childs {
+  padding: 1px 0.5rem;
+}
+
+.main__button {
+  font-size: 1rem;
+  font-weight: 700;
+  background-color: #ff5438;
+  color: #fff;
+  border: none;
+  padding: 1.5rem 3rem;
+  margin: 1rem 0;
+}
+
+.speakers {
+  display: flex;
+  gap: 15px;
+  padding: 4%;
+}
```

Comment:
- [ ] Great job using Javascript to dynamically generate the speaker's section. However, I noticed that you're missing the short horizontal line that comes after the speaker's name and title and separates them both from the speaker's description  have attached images below for your reference :+1: 
 - [ ] I also noticed that the width of each speaker when on a desktop screen is wider than that of the original design. Please try to ensure that the layout of your projects looks similar to that of the original design :100: 
 - Your Design
 
![image](https://user-images.githubusercontent.com/102699958/226071657-955b1f2e-140f-4928-8f1a-d171eb8943a3.png)

 - The Original Design
 
![image](https://user-images.githubusercontent.com/102699958/226071626-44b3f3d4-1931-440a-8a6b-fa39cfa4b9ef.png)